average.js

```diff
@@ -1,6 +1,24 @@
-
-function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+function average(numbers){
+    var avg = 0;
```

Comment:
`avg` is used at the end of this function. A good practice is to declare variables close to their usage, to reduce load on readers of the code. Can you declare this at the end, where it is being used?

Reply:
done